domains/botanybase.is-local.org.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "A simple website for simple Plant Identification!",
+
+    "domain": "is-not-a.dev",
+    "subdomain": "example",
```

Comment:
This subdomain right here needs to be edited.

Reply:
Aight, I'll make changes, thanks!